### DISCUSSION TITLE
Requesting test funds - 5Equjw221oPGnrNjd3NLEZVYZCiKwtupU8ta1HaubkSsugrr

### DISCUSSION THREAD

**Discussion by dwapetra** [entropyxyz/community#54] [Get Test Funds] [answered]
### Address

5Equjw221oPGnrNjd3NLEZVYZCiKwtupU8ta1HaubkSsugrr

**Comment by johnnymatthews** [comment, marked as answer]
This address doesn't seem to be a valid Substrate/Entropy address. Can you check that you copied your address from the CLI correctly?

If the address is 100% correct, then that's something we'll have to look into.

**Reply by dwapetra** [reply]
> This address doesn't seem to be a valid Substrate/Entropy address. Can you check that you copied your address from the CLI correctly?
> 
> If the address is 100% correct, then that's something we'll have to look into.

Yes, i think it is correct, i checked it a few times

<img width="471" alt="Снимок экрана 2024-06-20 в 18 55 08" src="https://github.com/entropyxyz/community/assets/146518610/d5e31eb0-6cbe-4c63-abe3-dd2880fe1b9a">

**Reply by johnnymatthews** [reply]
Well this is interesting. The Substrate Portal we're using to transfer funds usually does a good job of validating the address before sending anything. For some reason, the address you have here isn't registering as a proper address. I'm checking to see if this is a problem with the portal, or with the CLI.

In the meantime, can you create a new address using the CLI and send that to me please? Thanks!

**Reply by dwapetra** [reply]
yes, this is my new address - 5CrJBFn9UMYXzaYYiFcZFgr5Bv8g3ztBy9LYXQ2wSvP4SX46

**Reply by johnnymatthews** [reply]
Perfect, thanks for that. I've just sent some test funds to your account. I'm gonna close this conversation and mark it as answered. I'll create an issue in the CLI repo regrading the invalid address generation.

**Reply by dwapetra** [reply]
Great, thanks!
